libc/src/stdlib/quick_exit.h

```diff
@@ -0,0 +1,20 @@
+//===-- Implementation header for quick_exit -------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_LIBC_SRC_STDLIB_QUICK_EXIT_H
+#define LLVM_LIBC_SRC_STDLIB_QUICK_EXIT_H
+
+#include <stdlib.h>
```

Comment:
Do we need anything from stdlib to satisfy this declaration? I don't think so. Can you remove it?